modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -713,6 +713,77 @@ definition {
 		}
 	}
 
+	@description = "LPS-147651 - Verify that the user can define and use a expression with 'Is Empty' to text fields"
+	@priority = "4"
+	test CanDefineExpressionIsEmpty {
+		property portal.upstream = "quarantine";
+
```

Comment:
@andreefarias in the future, you should quarantine the test in a separate commit so it's easier to track the history of the test. Also, please create a ticket to remove the test from quarantine and link it to the automation ticket.